policies/secpolicy.html

```diff
@@ -38,21 +38,44 @@ <h2>Issue triage</h2>
 
 	    </p>
 
+	    <h2>Threat Model</h2>
+
+            <p>Certain threats are currently considered outside of the scope of the OpenSSL threat model.
+              Accordingly, we do not consider OpenSSL secure against the following classes of attacks:</p>
+              <ul>
+                <li>same physical system side channel</li>
+                <li>CPU/hardware flaws</li>
+                <li>physical fault injection</li>
+                <li>power consumption observations</li>
```

Comment:
EM emissions?

and I'm guessing that the sound side-channel falls under power consumption?

Reply:
Would changing the final item to _physical observation side channels_ be better?
Or _physical observation side channels (e.g. power consumption, EM emissions, etc)_ be better?

Reply:
yes, I like the latter more (though I'm not sure if combining "e.g." with "etc." is stylistically acceptable)

Reply:
@kroeckx is this change likely to be included?